v3/docs/02-concepts/h-off-chain-features/index.mdx

```diff
@@ -1,39 +1,34 @@
 ---
 title: Off-Chain Features
 slug: /v3/concepts/off-chain-features
-version: 3.0 
+version: 3.0
 section: docs
 category: concepts
 keywords: off-chain, off-chain storage, oracles, ocw
 ---
 
-There is often a need to query and/or process off-chain data before it can be included in the
-on-chain state. The conventional way of doing this is through oracles. Oracles are external services
-that typically listen to blockchain events and trigger tasks accordingly. When these tasks complete,
-their results are submitted back to the blockchain using transactions. While this approach works, it
-still has several flaws with respect to security, scalability, and infrastructure efficiency.
+There is often a need to query and/or process off-chain data before it can be included in the on-chain state.
+The conventional way of doing this is through [oracles](/v3/getting-started/glossary#oracle).
+While this approach works, it still has several flaws with respect to security, scalability, and infrastructure efficiency.
 
-To make the off-chain data integration secure and more efficient, Substrate provides off-chain
-features:
+To make the off-chain data integration secure and more efficient, Substrate provides off-chain features:
 
-- **Off-Chain Worker (OCW)** subsystem allows execution of long-running and possibly non-
-  deterministic tasks (e.g. web requests, encryption/decryption and signing of data, random number
-  generation, CPU-intensive computations, enumeration/aggregation of on-chain data, etc.) that could
-  otherwise require longer than the block execution time.
+- **Off-Chain Worker (OCW)** subsystem allows execution of long-running and possibly non- deterministic tasks
+  (e.g. web requests, encryption/decryption and signing of data, random number generation,CPU-intensive computations, enumeration/aggregation of on-chain data, etc.)
+  that could otherwise require longer than the block execution time.
```

Comment:
```suggestion
- **Off-Chain Worker (OCW)** subsystems allows execution of long-running and possibly non-deterministic tasks (e.g. web requests, encryption/decryption and signing of data, random number generation, CPU-intensive computations, enumeration/aggregation of on-chain data, etc.) that could otherwise require longer than the block execution time.
```

Reply:
One-line-ified.